package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Command line interface for JavaAutolab",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha test"
+    "lint": "jshint **.js",
```

Comment:
we do both eslint and jslint. both need to be here.
The js files are in different directories, not only in the top-level.